app/backend/approaches/chatreadretrieveread.py

```diff
@@ -381,13 +394,6 @@ async def run(self, history: Sequence[dict[str, str]], overrides: dict[str, Any]
             #print("System Message Tokens: ", self.num_tokens_from_string(system_message, "cl100k_base"))
             #print("Few Shot Tokens: ", self.num_tokens_from_string(self.response_prompt_few_shots[0]['content'], "cl100k_base"))
             #print("Message Tokens: ", self.num_tokens_from_string(message_string, "cl100k_base"))
-                chat_completion= await self.client.chat.completions.create(
```

Comment:
Is that not the same as the creation later on in the script that is being used? The later chat_completion should overwrite this since it is outside of the if and elif blocks. I believe the only thing changing is the messages variable between the two.

Reply:
Will flatten code to update unneeded extra calls as they can be the same now.